en/concept/sd_card_layout.md

```diff
@@ -0,0 +1,20 @@
+# PX4 SD Card Layout
+
+The PX4 SD Card is used for storing configuration files, flight logs, mission information etc. 
+
+> **Note** The file system is FAT32.
```

Comment:
Maybe note that this is the default, and you don't have to format before using it?

Reply:
Yes. That implies you can use other formats if you like. I've updated, but if we can change formats, we should probably link to how/where?

Reply:
Sorry, what do you mean by that?

Reply:
> Sorry, what do you mean by that?

So you said "Maybe note that this is the default". If something has a default of FAT32, that implies that you might have another file system. Right?

If so, what are the other options for a file system, and how do we set PX4 to use them?

Reply:
I see. You can format an SD card to any file system you want. But that's your conscious choice then and you have to make sure you switch back to FAT32 again if you want to use it with PX4 (we don't really support others).

Reply:
Thanks. So I had assumed that when you said "formatting is not required before use" the card will be formatted with FAT32 automatically by PX4 if it wasn't already. Is that the case.

If not, we should just say use a formatted FAT32 SD card, and ignore any nuances.

Reply:
There's logic to reformat the SD card if mouting fails, but I rather suggest users to format themselves (I'm not sure how reliable NuttX does it).

Reply:
I agree. Done in https://github.com/PX4/Devguide/pull/980.

Thanks.